kits/arms/default_3DOF_arm_gains.xml

```diff
@@ -1,52 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Were these not used anywhere?  What is the current approach for loading gains for the demo arms?